nixos/modules/services/web-apps/plausible.nix

```diff
@@ -277,7 +277,11 @@ in {
               ''export SMTP_USER_PWD="$(< $CREDENTIALS_DIRECTORY/SMTP_USER_PWD )"''}
 
             # setup
-            ${cfg.package}/createdb.sh
+            ${
+              if cfg.database.postgres.setup
```

Comment:
i would suggest `optionalString` here instead, but yeah this change looks right to me

@Ma27 - would you like to comment before merge?

Reply:
Updated to use `optionalString`; thanks for the tip!

Hopefully that looks better now :+1:

Reply:
awesome! thank you